docs/tasks/timeSeriesOHCAnomaly.rst

```diff
@@ -87,6 +87,11 @@ depth integrated time series will be read in with a file prefix given by
 ``preprocessedFieldPrefix``.  Generally, these options should not be altered
 except of debugging purposes.
 
+Recently, a right-hand axis and an associated set of lines has been added to the
+OHC anomaly time series.  This axis and these lines shows the equivalent
+top-of-atmosphere energy flux (:math:`W/m^2`) that the ocean heat anomaly would
+induce.
+
```

Comment:
@golaz, can you let me know if this is accurate or suggest changes if not?

Reply:
just change to 'show'.

Reply:
Sorry, I missed this before merging.  I'll fix it in #770